### ISSUE TITLE
Table: The table can be focused when the `minWidth` prop is set.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2277]
**Describe the bug**
The table can be focused when the `minWidth` prop is set.

**To Reproduce**
https://react-magma.cengage.com/version/4.11.0/api/table/#min-width

**Expected behavior**
The table should not be focused when the `minWidth` prop is set.

[Slack request](https://cengage.slack.com/archives/C8Y4FR63U/p1772099481957349)

The table should not be focused when the minWidth prop is set. --> Just another quick question, I think it should be focused if there are no other focusable elements inside. So the behavior https://react-magma.cengage.com/version/4.11.0/api/table/#min-width here is correct. It is only incorrect if there are some other elements inside that can receive focus.
Is it correct?

**Comment by silvalaura** [comment]
Double check this behavior, it may be functioning as designed. Will just need to communicate with mts.

**Comment by olehnoskov-cengage** [comment]
**Tested on:**
- V4:
Docs: https://next--upbeat-sinoussi-f675aa.netlify.app/api/table/

- V5: 
Docs: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/table/

Everything works as expected. No bugs found.
